### ISSUE TITLE
spells duration should be standardized

### ISSUE THREAD

**Issue by TSPARR** [ZachMyers3/flask-dnd#2]

**Comment by ZachMyers3** [comment]
- Spells with options have different duration (see below example)
- Make instantaneous 0 seconds
- Convert hours to seconds (?)
- Convert days to seconds (?)

Not sure the best way to represent this

```json
   {
    "name": "Mold Earth",
    "desc": "<p>You choose a portion of dirt or stone that you can see within range and that fits within a 5-foot cube. You manipulate it in one of the following ways:</p><ul><li>If you target an area of loose earth, you can instantaneously excavate it, move it along the ground, and deposit it up to 5 feet away. This movement doesn't have enough force to cause damage.</li><li>You cause shapes, colors, or both to appear on the dirt or stone, spelling out words, creating images, or shaping patterns. The changes last for 1 hour.</li><li>If the dirt or stone you target is on the ground, you cause it to become difficult terrain. Alternatively, you can cause the ground to become normal terrain if it is already difficult terrain. This change lasts for 1 hour.</li></ul><p>If you cast this spell multiple times, you can have no more than two of its non-instantaneous effects active at a time, and you can dismiss such an effect as an action.</p>",
    "book": "Elemental Evil Player's Companion",
    "page": 21,
    "range": "30 feet",
    "components": {
      "verbal": false,
      "somatic": true,
      "material": false,
      "materials_needed": "",
      "raw": "S"
    },
    "ritual": false,
    "duration": "Instantaneous/1 hour",
    "concentration": false,
    "casting_time": "Action",
    "level": 0,
    "school": "Transmutation",
    "classes": [
      "Druid",
      "Sorcerer",
      "Wizard"
    ]
  }
```

**Comment by ZachMyers3** [comment]
Fixed in [fc7300c](https://github.com/ZachMyers3/flask-dnd/commit/fc7300c36b02e0c121a0c133df729b2a9ce31cbd)
